cross/build-rootfs.sh

```diff
@@ -180,6 +180,7 @@ if [[ "$__LinuxCodeName" == "alpine" ]]; then
       -X http://dl-cdn.alpinelinux.org/alpine/v$__AlpineVersion/main \
       -X http://dl-cdn.alpinelinux.org/alpine/v$__AlpineVersion/community \
       -X http://dl-cdn.alpinelinux.org/alpine/edge/testing \
+      -X http://dl-cdn.alpinelinux.org/alpine/edge/main \
```

Comment:
> This is a copy of the same change made in dotnet/arcade.

I don't see this in Arcade:

https://github.com/dotnet/arcade/blob/b21c24996a73aa62b7a1ee69f546b9d2eb084f29/eng/common/cross/build-rootfs.sh#L202-L206

```
    $__ApkToolsDir/apk-tools-$__ApkToolsVersion/apk \
      -X http://dl-cdn.alpinelinux.org/alpine/v$__AlpineVersion/main \
      -X http://dl-cdn.alpinelinux.org/alpine/v$__AlpineVersion/community \
      -X http://dl-cdn.alpinelinux.org/alpine/edge/testing \
      -U --allow-untrusted --root $__RootfsDir --arch $__AlpineArch --initdb \
```

Maestro++ does seem to be updating the eng/common copy of this file fine:

https://github.com/dotnet/core-setup/blob/7b45d027a2ba812dc575df9a98162ca91fa1b828/eng/common/cross/build-rootfs.sh#L202-L206

Reply:
My mistake, I had an old checkout on my machine before the eng was added. It is really there.
As for the arcade repo, my change was not merged in yet.

Reply:
Ah, I see, I took `made` the wrong way. 😄

In any case, the old files should be deleted from Core-Setup and things depending on them pointed to eng/common to make this clearer. (But no need to do that in this PR.)

Reply:
Right, we need to eventually do that, for coreclr repo too. For corefx, such a change is pending a PR. I just don't have enough time now for fiddling with the build scripts.